reference/03-Blocks/unregister-block.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 10
+---
+
+# Unregister a Block
+
+## Reference Guide WordPress Documentation:
+* [unregisterBlockType](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-blocks/#unregisterblocktype)
+
+## Use Case:
+Gutenberg comes with a lot of core blocks. When creating custom builds for clients all of these core blocks are generally not needed nor should they be used by the client. In these situations it is necessary to disable the blocks from the block editor so that a user can not access them when creating new pages and posts with the block editor. This can be done using the `unregisterBlockType` function and passing in the blocks which should be disallowed within a `.js` file. 
+
+
+## Code Example:
+This example showcases how an array of blocks can be passed into a single function and looped through to disable multiple blocks at the same time. 
+
+`block-filters/unregister-blocks.js`
+```bash
+const { unregisterBlockType } = wp.blocks;
```

Comment:
We tend to use the import syntax in the documentation here as it is standard on all modern builds :) 

```suggestion
import { unregisterBlockType } from '@wordpress/blocks';
```